Zend/zend_exceptions.stub.php

```diff
@@ -48,7 +48,7 @@ public function __toString(): string {}
 
 class ErrorException extends Exception
 {
-    public function __construct(string $message = "", int $code = 0, int $severity = E_ERROR, ?string $filename = null, ?int $line = null, ?Throwable $previous = null) {}
+    public function __construct(string $message = "", int $code = 0, int $error_level = E_ERROR, ?string $filename = null, ?int $line = null, ?Throwable $previous = null) {}
```

Comment:
`$severity` seems to be an outlier, but there's a property is called the same, so it's something to think about.

Reply:
Hm... in that case, I've stick to `$severity`. It makes sense the arguments have the same name as the property they initialize.

Reply:
Yes, i agree, so reverted this one. My motivation for changing this was mainly to provoke some discussion. :)